src/components/create-project-form/index.js

```diff
@@ -12,67 +16,79 @@ import {
   ProjectDescriptionInput,
   ProjectCategoryInput,
   ProjectImpactLocationInput,
-  ProjectImageInput
+  ProjectImageInput,
+  ProjectEthAddressInput
 } from './inputs'
-import { CloseModal } from './modals'
 import EditButtonSection from './EditButtonSection'
 import FinalVerificationStep from './FinalVerificationStep'
+import ConfirmationModal from '../confirmationModal'
+import { categoryList } from '../../utils/constants'
```

Comment:
Earmarking this as well - remember we decided to curate categories ourselves. We should start coming up with the ones we want to use in MVP

Reply:
Yup, that's why I moved to a constants file so the change can be quick. Let's start the discussion on the next meeting